Privacy: Expand Compose, revise Copilot (#3206)
MicrosoftDocs/edge-developer: microsoft-edge/privacy-whitepaper/index.md

<!-- ====================================================================== -->
## Cast

Cast in Microsoft Edge lets you display your media to another screen using Google Cast.  To access Cast, open **Settings and more (...)** > **More tools** > **Cast media to device**.  Cast relies on the Media Router extension not automatically installed with Microsoft Edge.  When you first use Cast, Microsoft Edge prompts for permission to install the Media Router extension.

> Select **restart** to install the Media Router extensions from the Chrome Web Store. On startup and at regular intervals, Microsoft Edge sends update requests to the Chrome Web Store. Update requests include basic data about your version of Microsoft Edge. Regular update requests keep the Media Router extension up to date.

Reviewer: Cast h2 section is out of scope for PR.  Closing.  Can handle in a future PR, that could be for whole whitepaper.
@majenny @apryzgoda

> Select **restart** to install the Media Router extensions from the Chrome Web Store. On startup and at regular intervals, Microsoft Edge sends update requests to the Chrome Web Store. Update requests include basic data about your version of Microsoft Edge. Regular update requests keep the Media Router extension up to date.

Reviewer: PR moves Copilot to below Cast, Collections, & Compose.  That caused all those sections to be listed in Files Changed as if changed.  PR is really about Compose, and then added Copilot b/c that's recently added & needed rewording.

> Select **restart** to install the Media Router extensions from the Chrome Web Store. On startup and at regular intervals, Microsoft Edge sends update requests to the Chrome Web Store. Update requests include basic data about your version of Microsoft Edge. Regular update requests keep the Media Router extension up to date.

Reviewer: made this change

> Google may collect some data associated with the Media Router extension. To uninstall the Media Router extension, go to `edge://flags#edge-on-demand-media-router` and change the setting. Uninstalling also stops updates from the Chrome Web Store. The extension is hidden and does not appear on the **Installed extensions** list. For the **Installed extensions** list, go to `edge://extensions`.

Reviewer: result:
```
go to `edge://flags#edge-on-demand-media-router`
<!-- todo: returns list of flags that doesn't include "Edge 
on demand media router".  related flag: "Connection to 
Cast devices on all IP addresses" --> and disable the feature.
```

> Google may collect some data associated with the Media Router extension. To uninstall the Media Router extension, go to `edge://flags#edge-on-demand-media-router` and change the setting. Uninstalling also stops updates from the Chrome Web Store. The extension is hidden and does not appear on the **Installed extensions** list. For the **Installed extensions** list, go to `edge://extensions`.

Reviewer: Cast h2 section is out of scope for this PR

When you sign in with a work or school account and Sync for Collections is disabled, all collections data is stored locally on the device, organized per Microsoft Edge profile.

You can access Collections from an InPrivate window while signed in with a work or school account.

<!-- ------------------------------ -->
#### Signed-out

When you don't sign in to Microsoft Edge, all collections data is stored locally on the device, organized per Microsoft Edge profile.

You can access Collections from an InPrivate window when using a signed-out profile.

<!-- ====================================================================== -->
## Compose

The Compose tool helps you draft text within the Microsoft Edge Sidebar, or rewrite selected text on any webpage.

<!-- ------------------------------ -->
#### Use Compose to draft text within the Edge Sidebar

In the Copilot pane, the Compose tool uses AI to write for you, starting from a short prompt and **Tone**, **Format**, and **Length** selections.

Text that's generated by Compose can be copied, or inserted directly into the webpage.

> To access the Compose tool, in the upper right of Microsoft Edge, click the **Copilot** (![Copilot icon](./index-images/copilot-icon.png)) button, and then select **Compose**.  Or, press **Ctrl+Shift+.**.  Text that you select or enter is sent to an internal Microsoft Large Language Model (LLM).

Reviewer 2: ```suggestion
To access the Compose tool, in the upper right of Microsoft Edge, click the **Copilot** (![Copilot icon](./index-images/copilot-icon.png)) button, and then select **Compose**.  Or, press **Ctrl+Shift+.**.  Text that you select or enter is sent to an internal Microsoft service that runs a Large Language Model (LLM).
```

> To access the Compose tool, in the upper right of Microsoft Edge, click the **Copilot** (![Copilot icon](./index-images/copilot-icon.png)) button, and then select **Compose**.  Or, press **Ctrl+Shift+.**.  Text that you select or enter is sent to an internal Microsoft service that runs a Large Language Model (LLM).

Reviewer: finally thought of good/smooth alt; changed to:
```
Or, use the **Ctrl+Shift+.** keyboard shortcut.
```

> To access the Compose tool, in the upper right of Microsoft Edge, click the **Copilot** (![Copilot icon](./index-images/copilot-icon.png)) button, and then select **Compose**.  Or, press **Ctrl+Shift+.**.  Text that you select or enter is sent to an internal Microsoft service that runs a Large Language Model (LLM).

Reviewer 3: ```suggestion
To access the Compose tool, in the upper right of Microsoft Edge, click the **Copilot** (![Copilot icon](./index-images/copilot-icon.png)) button, and then select **Compose**.  Or, press **Ctrl+Shift+.**. 
```
This is already noted in the 'Privacy and data protections for Compose' section. Removing for clarity and conciseness.

Reviewer: Removed sentence, although that later section doesn't have that exact sentence.  has paragraph:
```
Text that you've highlighted or inputted for Compose 
is sent to Microsoft after you click the **Rewrite** or 
**Generate** button, not while you're selecting text.  Each 
request contains the input prompt text, formatting selections, 
and IDs that are associated with your Microsoft Edge profile 
and device, to reference for subsequent requests.  Requests 
for Compose are sent over a secure HTTPS connection.
```
so, after new commit, whitepaper won't contain "large language" at all.
@apryzgoda @majenny

To change settings for the sidebar or Copilot pane, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Sidebar** > **App and notification settings** > **Copilot**.

<!-- ------------------------------ -->
#### Use Compose to rewrite selected text on any webpage

The Compose tool can help rephrase what you've written, and iterate using the same **Tone**, **Format**, and **Length** selections. 

To access the Compose tool directly in a webpage, highlight text longer than five words in any input text field, and then click the **Rewrite with Copilot** button.  Or, press **Alt+I**.

The Compose tool might not be available on some webpages or for certain input fields.

> To disable the Compose tool for all webpages, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Languages**, and then turn off the **Collaborate with Copilot as you write on the web** toggle.

Reviewer: ```
To disable the Compose tool for all webpages, 
select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) 
> **Languages**, and then turn off the **Collaborate with 
Copilot as you write on the web** setting.
```

<!-- ------------------------------ -->
#### Privacy and data protections for Compose

Text that you've highlighted or inputted is sent to Microsoft after you click the **Rewrite** or **Generate** button, not while you're selecting text.  Each request contains the input prompt text, formatting selections, and IDs that are associated with your Microsoft Edge profile and device, to reference for subsequent requests.  Requests for Compose are sent over a secure HTTPS connection.

For enterprises, additional options for data protection are available for Compose.  See [Privacy and protections](/copilot/privacy-and-protections) in the Microsoft Copilot documentation.

<!-- ====================================================================== -->
## Copilot in Microsoft Edge sidebar

In Copilot in the Microsoft Edge sidebar, you can ask complex questions, find comprehensive answers, get summarized information, and find inspiration to build upon, in a side-by-side view, with no need to flip between tabs.

> To open Copilot in the Microsoft Edge sidebar, click the **Copilot** (![Copilot icon.](./index-images/copilot-icon.png)) button (**Ctrl+Shift+.**) in the upper right of the Microsoft Edge toolbar.

Reviewer 3: ```suggestion
To open Copilot in the Microsoft Edge sidebar, click the **Copilot** (![Copilot icon.](./index-images/copilot-icon.png)) button in the upper right of the Microsoft Edge toolbar, or press (**Ctrl+Shift+.**) .
```

Reviewer: result:
```
To open Copilot in the Microsoft Edge sidebar, click 
the **Copilot** (![Copilot icon.](./index-images/copilot-icon.png)) 
button in the upper right of the Microsoft Edge 
toolbar.  Or, use the **Ctrl+Shift+.** keyboard 
shortcut.
```

<!-- --------------------------------------------------------------------- -->
#### Data and consent used by Copilot in Microsoft Edge 

Microsoft Edge determines what data to send to Copilot based on your **query** and your consent to share data with Microsoft.  For questions that don't need **browsing context**, such as "Help me plan a trip to Cannon Beach", Microsoft Edge shares the URL, the page title, your query, and previous conversation history to help Copilot answer your query effectively.

For questions that need **browsing context**, such as when you ask Copilot to summarize a large page of text, Microsoft Edge asks for your permission to access page information:

![Copilot asking for consent to access the page content in Microsoft Edge](./index-images/consent-for-contextual-browsing.png) <!-- Update Bing chat screenshot-->

When you grant permission to share page information, Microsoft Edge sends the full **browsing context** in addition to your **query** and previous **conversation history** to Copilot, to help generate a meaningful **response**.

> To modify this permission, go to Microsoft Edge > **Settings** > **Sidebar** > **App and notification settings** > **App specific settings** > **Copilot**, and then turn on or off the **Allow access to any webpage or PDF** toggle.

Reviewer 3: ```suggestion
To modify this permission, go to Microsoft Edge > **Settings** > **Sidebar** > **App and notification settings** > **App specific settings** > **Copilot**, and then toggle the **Allow access to any webpage or PDF** setting.
```

Reviewer: Apple style guide: 
https://support.apple.com/guide/applestyleguide/t-apsg841c3645/1.0/web/1.0#apd37a491bb67054 - "_toggle_ (as verb): Don’t use in user materials; instead, say turn on or off, switch between, or whatever wording is appropriate in the context."

<!-- ====================================================================== -->
## Compose

<!-- ------------------------------ -->
#### Draft text by using Compose in the Edge Sidebar

In the Copilot pane, the Compose tool uses AI to write for you, starting from a short prompt and **Tone**, **Format**, and **Length** selections:

![Compose UI](./index-images/compose.png)

Text that's generated by Compose can be copied, or inserted directly into the webpage.

To access the Compose tool, in the upper right of Microsoft Edge, click the **Copilot** (![Copilot icon](./index-images/copilot-icon.png)) button (**Ctrl+Shift+.**), and then select **Compose**.  Text that you select or enter is sent to an internal Microsoft Large Language Model (LLM). 

To change settings for the sidebar or Copilot pane, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Sidebar** > **App and notification settings** > **Copilot**.

<!-- ------------------------------ -->
#### Rewrite selected text on any webpage by using Compose 

> The Compose tool can also help rephrase what you've written, and iterate using the same **Tone**, **Format**, and **Length** selections. 

Reviewer: get rid of "also".  instead, in the h2 section / intro, list or mention both scenarios of Compose: eg:

The Compose tool enables you to draft text by using Compose in the Edge Sidebar, or rewrite selected text on any webpage by using Compose.

i can enter specific Suggested Change comments, or push a commit to impl. this

Reviewer: ```
## Compose
The Compose tool helps you draft text within the Microsoft Edge Sidebar, or rewrite selected text on any webpage.

#### Use Compose to draft text within the Edge Sidebar
In the Copilot pane, the Compose tool uses AI to write for you, starting from a short prompt and **Tone**, **Format**, and **Length** selections: ...

#### Use Compose to rewrite selected text on any webpage
The Compose tool can help rephrase what you've written, and iterate using the same **Tone**, **Format**, and **Length** selections. ...
```

<!-- ====================================================================== -->
## Compose

The Compose tool helps you draft text within the Microsoft Edge Sidebar, or rewrite selected text on any webpage.

<!-- ------------------------------ -->
#### Use Compose to draft text within the Edge Sidebar

> In the Copilot pane, the Compose tool uses AI to write for you, starting from a short prompt and **Tone**, **Format**, and **Length** selections:

Reviewer 3: ```suggestion
In the Copilot pane, the Compose tool uses AI to write for you, starting from a short prompt and **Tone**, **Format**, and **Length** selections.
```

Reviewer: a minor change that could go either way.  check rendered view re: flow.  often in this repo, we use colon to glue the lead-in sentence to the subsequent png that that sentence leads into or introduces.

Reviewer: subseq comment removes png, so no lead-in colon

> ![Compose UI](./index-images/compose.png)

Reviewer 3: not relevant to privacy
```suggestion
```

Reviewer: removed colon, png line, & png file.  result:
```
In the Copilot pane, the Compose tool uses AI 
to write for you, starting from a short prompt 
and **Tone**, **Format**, and **Length** selections.

Text that's generated by Compose can be copied, 
or inserted directly into the webpage.
```

Text that's generated by Compose can be copied, or inserted directly into the webpage.

> To access the Compose tool, in the upper right of Microsoft Edge, click the **Copilot** (![Copilot icon](./index-images/copilot-icon.png)) button (**Ctrl+Shift+.**), and then select **Compose**.  Text that you select or enter is sent to an internal Microsoft Large Language Model (LLM). 

Reviewer: separated the two ways.  result:
```
To access the Compose tool, in the upper right of Microsoft Edge, 
click the **Copilot** (![Copilot icon](./index-images/copilot-icon.png)) 
button, and then select **Compose**.  Or, press **Ctrl+Shift+.**.  Text 
that you select or enter is sent to an internal Microsoft Large 
Language Model (LLM).
```

To change settings for the sidebar or Copilot pane, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Sidebar** > **App and notification settings** > **Copilot**.

<!-- ------------------------------ -->
#### Use Compose to rewrite selected text on any webpage

The Compose tool can help rephrase what you've written, and iterate using the same **Tone**, **Format**, and **Length** selections. 

> To access the Compose tool directly in a webpage, highlight text longer than five words in any input text field, and then click the **Rewrite with Copilot** button (**Alt+I**):

Reviewer: removed colon, png line, & png.  Result:
```
To access the Compose tool directly in a webpage, highlight text longer than five words 
in any input text field, and then click the **Rewrite with Copilot** button.  Or, press **Alt+I**.

The Compose tool might not be available on some webpages or for certain input fields.
```

> ![The Compose UI](./index-images/compose-inline.png)

Reviewer 3: ```suggestion
```
Not relevant to privacy

Reviewer: removed colon, png line, & png.  Result:
```
To access the Compose tool directly in a webpage, highlight text longer than 
five words in any input text field, and then click the **Rewrite with Copilot** 
button.  Or, press **Alt+I**.

The Compose tool might not be available on some webpages or for certain input fields.
```

The Compose tool might not be available on some webpages or for certain input fields.

> Text that you've highlighted or inputted are sent to Microsoft only after you click the **Rewrite** or **Generate** button, not while you are selecting text.  No additional webpage data is ever shared.  Requests for Compose are sent over a secure HTTPS connection.  Each request contains the input prompt text, formatting selections, and IDs that are associated with your Microsoft Edge profile and device, to reference for subsequent requests.

Reviewer: I made that change; at present, still mentions Generate button in this h4 section, and claims that the ID is sent - other reviewer might be questioning those two aspects, in other Comments.  @majenny  - see fresh Files Changed tab for latest commits/Markdown.

result at the moment: 
```
Text that you've highlighted or inputted is sent to Microsoft 
after you click the **Rewrite** or **Generate** button, 
not while you're selecting text.  Each request contains 
the input prompt text, formatting selections, and IDs 
that are associated with your Microsoft Edge profile and 
device, to reference for subsequent requests.  Requests for 
Compose are sent over a secure HTTPS connection.
```

> To disable the Compose tool for all webpages, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Languages**, and then turn off the toggle **Collaborate with Copilot as you write on the web**.

Reviewer: corp style: 
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/t/toggle
result: 
```
To disable the Compose tool for all webpages, select 
**Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) 
> **Languages**, and then turn off the **Collaborate with 
Copilot as you write on the web** toggle.
```

For enterprises, additional options for data protection are available for Compose.  See [Privacy and protections](/copilot/privacy-and-protections) in the Microsoft Copilot documentation.

<!-- ====================================================================== -->
## Copilot in Microsoft Edge sidebar

In Copilot in the Microsoft Edge sidebar, you can ask complex questions, find comprehensive answers, get summarized information, and find inspiration to build upon, in a side-by-side view, with no need to flip between tabs.

> To open Copilot in the Microsoft Edge sidebar, click the **Copilot** (![Copilot icon.](./index-images/copilot-icon.png)) icon (**Ctrl+Shift+.**) in the upper right of the Microsoft Edge toolbar: 

Reviewer: To open Copilot in the Microsoft Edge sidebar, click the **Copilot** (![Copilot icon.](./index-images/copilot-icon.png)) icon in the upper right of the Microsoft Edge toolbar.  Or, press (**Ctrl+Shift+.**).

> To open Copilot in the Microsoft Edge sidebar, click the **Copilot** (![Copilot icon.](./index-images/copilot-icon.png)) icon (**Ctrl+Shift+.**) in the upper right of the Microsoft Edge toolbar: 

Reviewer: rendered section:
https://review.learn.microsoft.com/en-us/microsoft-edge/privacy-whitepaper/?branch=pr-en-us-3206#copilot-in-microsoft-edge-sidebar

> ![Summarize this page](./index-images/summarize-content.png) 

Reviewer: rendered section:
https://review.learn.microsoft.com/en-us/microsoft-edge/privacy-whitepaper/?branch=pr-en-us-3206#copilot-in-microsoft-edge-sidebar

> ![Summarize this page](./index-images/summarize-content.png) 

Reviewer: ```
To open Copilot in the Microsoft Edge sidebar, 
click the **Copilot** (![Copilot icon.](./index-images/copilot-icon.png)) 
button (**Ctrl+Shift+.**) in the upper right of the Microsoft Edge toolbar.

<!-- --------------------------------------------------------------------- -->
```

<!-- --------------------------------------------------------------------- -->
#### Data and consent used by Copilot in Microsoft Edge 

Microsoft Edge determines what data to send to Copilot based on the user's **query** and their consent to share data with Microsoft.  For questions that don't need **browsing context**, such as "Help me plan a trip to Cannon Beach", Microsoft Edge shares the URL, page title, user's query, and previous conversation history to help Copilot answer their query effectively.

For questions that need **browsing context**, such as when the user asks Copilot to summarize a large page of text, Microsoft Edge requests permission from the user to access page information:

![Copilot asking for consent to access the page content in Microsoft Edge](./index-images/consent-for-contextual-browsing.png) <!-- Update Bing chat screenshot-->

When the user grants permission to share page information, Microsoft Edge sends the full **browsing context** in addition to the user's **query** and previous **conversation history** to Copilot, to help generate a meaningful **response**.

> The user can modify this permission anytime by going to Microsoft Edge > **Settings** > **Sidebar** > **App and notification settings** > **App specific settings** > **Copilot** and then turning on or off the **Allow access to any webpage or PDF** toggle.

Reviewer: as an option, corp style, not necessarily binding:
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/t/toggle

> The user can modify this permission anytime by going to Microsoft Edge > **Settings** > **Sidebar** > **App and notification settings** > **App specific settings** > **Copilot** and then turning on or off the **Allow access to any webpage or PDF** toggle.

Reviewer: seems like section needs to change "the user" to "you" per corp style

> The user can modify this permission anytime by going to Microsoft Edge > **Settings** > **Sidebar** > **App and notification settings** > **App specific settings** > **Copilot** and then turning on or off the **Allow access to any webpage or PDF** toggle.

Reviewer: rendered article section:
https://review.learn.microsoft.com/en-us/microsoft-edge/privacy-whitepaper/?branch=pr-en-us-3206#data-and-consent-used-by-copilot-in-microsoft-edge

> The user can modify this permission anytime by going to Microsoft Edge > **Settings** > **Sidebar** > **App and notification settings** > **App specific settings** > **Copilot** and then turning on or off the **Allow access to any webpage or PDF** toggle.

Reviewer: Survey of use of 'toggle' in whitepaper shows need for global correction of usage of 'toggle', to match corp style & for consistency, to make maintenance easier: this survey covers "A" through "Image descriptions":
* Toggle the **Show me search and site suggestions using my typed characters** setting. - incorrect per corp style.
* Select **Personal info** and toggle the **Save and fill personal info** setting. - incorrect.
* Toggle off the **Offer to save passwords** setting.  - incorrect.
* Go to `edge://settings/profiles/sync` and turn off the **Basic info** toggle. - correct (other than special url instead of GUI nav path).
* If you want to disable Collections for a signed-in profile using a personal account, go to `edge://settings/privacy`, scroll down to the **Services** section, and then turn off the toggle for **Show Collections and follow content creators in Microsoft Edge**. - correct.
* To disable the Compose tool for all webpages, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Languages**, and then turn off the toggle **Collaborate with Copilot as you write on the web**. - correct 
* The user can modify this permission anytime by going to Microsoft Edge > **Settings** > **Sidebar** > **App and notification settings** > **App specific settings** > **Copilot** and then turning on or off the **Allow access to any webpage or PDF** toggle. - correct except "the user" should be "you", and should "to do x, turn ..." not "by turning".
* Toggle the **Allow sites to play protected content (recommended)** and **Allow identifiers for protected content (computer restart may be required)** settings. - incorrect.
* To change SmartScreen settings, go to `edge://settings/privacy` and toggle SmartScreen. - incorrect, and "SmartScreen" should be bold & match UI label.
* Turn off the **From other sources** toggle to uninstall extensions. - ok but sentence should be reversed: To do x, turn off.
* Toggle the **Allow extensions from other stores** setting. - incorrect.
* When the **Include related matches in Find on page** toggle is turned on, Microsoft Edge sends the text of the webpage, your search terms, and a service token to a Microsoft cloud service over a secure HTTPS connection.  - correct.
* 1. Turn off the **Include related matches in Find on page** toggle. - correct.
* Or, go to `edge://settings/privacy`, scroll down to the **Services** section, and then turn off the toggle for **Save screenshots of site for History**.  - correct (except special url should be GUI nav path).
* Toggle the **Get image descriptions from Microsoft for screen readers** setting.  - incorrect.

> The user can modify this permission anytime by going to Microsoft Edge > **Settings** > **Sidebar** > **App and notification settings** > **App specific settings** > **Copilot** and then turning on or off the **Allow access to any webpage or PDF** toggle.

Reviewer: result re: the Suggestion:

```
When you grant permission to share page information, Microsoft Edge sends the 
full **browsing context** in addition to your **query** and previous **conversation history** 
to Copilot, to help generate a meaningful **response**.

To modify this permission, go to Microsoft Edge > **Settings** > **Sidebar** > **App 
and notification settings** > **App specific settings** > **Copilot**, and then turn on or 
off the **Allow access to any webpage or PDF** toggle.
```

<!-- ---------------------------------------------------------------------- -->
#### Duration of storage of conversation history

* **Conversation history** is a history of all queries and responses from Copilot.  **Conversation history** is stored for 90 days.  This is independent of whether using Copilot in Bing or Copilot in the Microsoft Edge sidebar or by visiting `www.bing.com/chat`.

   * Users can delete a specific conversation or all the conversations from Copilot in Bing by going to `www.bing.com/chat`, and then deleting the specific conversations in the **Recent activity** section:

     ![Recent conversation history in Copilot](./index-images/bing-chat-recent.png)

> * **Page content** from public **pages on the Bing index** – Copilot uses publicly available information from the Bing search engine to summarize pages that are available on the Bing search index.  To answer questions from public pages, Microsoft Edge doesn't need to share this page content with Copilot.

Reviewer: rendered article section:
https://review.learn.microsoft.com/en-us/microsoft-edge/privacy-whitepaper/?branch=pr-en-us-3206#duration-of-storage-of-conversation-history

<!-- ====================================================================== -->

> ## Compose

Reviewer: need to have some general text directly in this h2 section.  what can we say about Compose tool/feature in general, that's not specific to the detailed Sidebar or Rewrite h4 sections?  @majenny

Reviewer: ```
## Compose
The Compose tool helps you draft text within the Microsoft Edge Sidebar, or rewrite selected text on any webpage.

#### Use Compose to draft text within the Edge Sidebar
In the Copilot pane, the Compose tool uses AI to write for you, starting from a short prompt and **Tone**, **Format**, and **Length** selections: ...

#### Use Compose to rewrite selected text on any webpage
The Compose tool can help rephrase what you've written, and iterate using the same **Tone**, **Format**, and **Length** selections. ...
```

In the Copilot pane, the Compose tool uses AI to write for you, starting from a short prompt and **Tone**, **Format**, and **Length** selections. Text that's generated by Compose can be copied or inserted into the webpage directly.

![Compose UI](./index-images/compose.png)

To access Compose, click the **Copilot** (![Copilot icon](./index-images/copilot-icon.png)) icon (**Ctrl+Shift+.**) in the upper right of Microsoft Edge, and then select **Compose**. Text and selections that you enter are sent to an internal Microsoft Large Language Model (LLM). 

To change settings for the sidebar or Copilot pane, go to `edge://settings/sidebar`, and then select **App and notification settings** > **Copilot**.

#### Rewrite on any webpage with Compose 
Compose can also help rephrase what you've written and iterate using the same **Tone**, **Format**, and **Length** selections. 

To access Compose directly in a webpage, highlight text longer than 5 words in any input text field. Click the "Rewrite with Copilot" button or Alt+I shortcut to start .  Compose may not be available on some webpages and certain input fields.

![Compose UI](./index-images/compose-inline.png)

> To disable Compose on all webpages, go to `edge://settings/languages`, and then toggle off **Collaborate with Copilot as you write on the web**. 

Reviewer: todo - you or i can do this.  get rid of the special edge:// url, give GUI nav path instead.  the wording pattern to use is in open PR https://github.com/MicrosoftDocs/edge-developer/pull/3127 (Separate "Privacy, search, and services" into "Privacy and security" & "Search and services").  I can provide Suggested Change comment soon, or i can push a commit.

![image](https://github.com/MicrosoftDocs/edge-developer/assets/45407486/07a73276-7bf1-4766-900f-9555e44d194b)

Reviewer: done 2x in the h2 section; pushed commit

> Text you've highlighted or inputted are sent to Microsoft only after "Rewrite" or "Generate" buttons are clicked, not while selections are being made. No additional webpage data is ever shared. Requests for Compose are sent over a secure HTTPS connection. Each request contains the input prompt, formatting selections, and IDs that are associated with your Microsoft Edge profile and device, to reference for subsequent requests.

Reviewer 2: Should the "generate" button be referenced in the "Draft with Compose in Edge Sidebar" section instead?

Reviewer: @majenny - not sure if button label is "Generate" - make sure the text matches control's actual label / tooltip.  
Removed some "how to use feature's UI" pngs per other review comments.

Author: Resolved! There are two experiences here, one with a "Rewrite" button and one with "Generate". I just created new section for Privacy that talks about both.

> Text you've highlighted or inputted are sent to Microsoft only after "Rewrite" or "Generate" buttons are clicked, not while selections are being made. No additional webpage data is ever shared. Requests for Compose are sent over a secure HTTPS connection. Each request contains the input prompt, formatting selections, and IDs that are associated with your Microsoft Edge profile and device, to reference for subsequent requests.

Reviewer 2: Do we really send "IDs that are associated with your Microsoft Edge profile and device"?

Author: I believe AnaheimIDs and DeviceIDs (?) are sent with each query as per our privacy whitepaper

Author: Just confirmed both are sent in [whitepaper](https://microsoft.sharepoint.com/:w:/t/Edge/ERovfJIhynVBmkpCq1Pba_EBSeTPoNAMq9xIceElLIiKPA?e=ryRWvd)

To access Compose, click the **Copilot** (![Copilot icon](./index-images/copilot-icon.png)) icon (**Ctrl+Shift+.**) in the upper right of Microsoft Edge, and then select **Compose**. Text and selections that you enter are sent to an internal Microsoft Large Language Model (LLM). 

Requests for Compose are sent over a secure HTTPS connection. Each request contains the input prompt, formatting selections, and IDs that are associated with your Microsoft Edge profile and device, to reference for subsequent requests.

To change settings for the sidebar or Copilot pane, go to `edge://settings/sidebar`, and then select **App and notification settings** > **Copilot**.

> #### Rewrite on any webpage with Compose 

Reviewer 2: We should rename the existing Compose section above to reflect that is just about "Compose sidebar".

Author: added!

Compose can also help rephrase what you've written and iterate using the same **Tone**, **Format**, and **Length** selections. 

> To access Compose directly in a webpage, highlight text longer than 5 words in any input text field. Click the "Rewrite with Copilot" button or Alt+I shortcut to start .  Compose may not be available on some webpages and certain input fields.

Reviewer 2: We should add details about data flow. Ie. Data is only sent to the model after clicking the tooltip, not when the user is making a selection.

Author: added!

> 

Reviewer 2: Should we also add details about the data protections for enterprises?

Author: added link!
